### ISSUE TITLE
KeyError: 'maxFuel'

### ISSUE THREAD

**Issue by roiff** [bimmerconnected/bimmer_connected#141]
**Describe the issue**
My model is 530LE 
Hybrid Electric
Sales in China

Following is the error log
```
2019-12-04 19:15:05 ERROR (MainThread) [homeassistant.components.sensor] bmw_connected_drive: Error on device update!
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 292, in _async_add_entity
    await entity.async_device_update(warning=False)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 450, in async_device_update
    await self.hass.async_add_executor_job(self.update)
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/homeassistant/homeassistant/components/bmw_connected_drive/sensor.py", line 148, in update
    self._state = getattr(vehicle_state, self._attribute)
  File "/usr/local/lib/python3.7/site-packages/bimmer_connected/state.py", line 307, in __getattr__
    return self._attributes[item]
KeyError: 'maxFuel'
````

**Comment by ronmichel** [comment]
see https://github.com/bimmerconnected/bimmer_connected/issues/129

**Comment by Quentame** [comment]
Duplacates #129

**Comment by gerard33** [comment]
Fixed in #148 and [HA PR](https://github.com/home-assistant/home-assistant/pull/31894) made.
